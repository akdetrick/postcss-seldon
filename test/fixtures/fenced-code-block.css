/**doc
---
name: FCB test
---

```js
const lol = 3;
```
*/
