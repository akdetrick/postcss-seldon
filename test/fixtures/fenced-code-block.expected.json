{
  "meta": {
    "name": "FCB test"
  },
  "markdown": "\n```js\nconst lol = 3;\n```",
  "html": "<pre><code class=\"language-js\">const lol = 3;</code></pre>\n"
}
